micro-rdk-nmea/src/lib.rs

```diff
@@ -157,10 +156,17 @@ mod tests {
         let res = general_purpose::STANDARD.decode_vec(msg_str, &mut data);
         assert!(res.is_ok());
 
-        let cursor = DataCursor::new(data[MESSAGE_DATA_OFFSET..].to_vec());
-        let message = GnssSatsInView::from_cursor(cursor);
-        assert!(message.is_ok());
+        let nmea_message = NmeaMessage::try_from(data);
+        assert!(nmea_message.is_ok());
+
+        let nmea_message = nmea_message.unwrap();
+        let message = match nmea_message.data {
+            NmeaMessageBody::GnssSatsInView(val) => Some(val),
+            _ => None,
+        };
+        assert!(message.is_some());
         let message = message.unwrap();
+
```

Comment:
@gvaradarajan - New since last review, this now ensures that the NmeaMessage enum gets tested at least once.

Reply:
thank you for adding this